src/api/index.test.js

```diff
@@ -215,17 +308,29 @@ it('getPipelineRuns With Query Params', () => {
     items: 'pipelineRuns'
   };
   fetchMock.get(/pipelineruns/, data);
-  return getPipelineRuns({ pipelineName, filters: [] }).then(pipelineRuns => {
-    expect(pipelineRuns).toEqual(data.items);
-    fetchMock.restore();
-  });
+  return index
+    .getPipelineRuns({ pipelineName, filters: [] })
+    .then(pipelineRuns => {
+      expect(pipelineRuns).toEqual(data.items);
+      fetchMock.restore();
+    });
 });
 
 it('getPipelineRun', () => {
   const name = 'foo';
   const data = { fake: 'pipelineRun' };
   fetchMock.get(`end:${name}`, data);
-  return getPipelineRun({ name }).then(pipelineRun => {
+  return index.getPipelineRun({ name }).then(pipelineRun => {
+    expect(pipelineRun).toEqual(data);
+    fetchMock.restore();
+  });
+});
+
+it('deletePipelineRun', () => {
+  const name = 'foo';
+  const data = { fake: 'pipelineRun' };
+  fetchMock.delete(`end:${name}`, data);
```

Comment:
Add a comment here so it's easier to understand/main please, what's end: do? Why's it now required?

Reply:
This is someones PR, i just added the `index.` before because had to add the index for jest to work